README.md

```diff
@@ -1,32 +1,27 @@
 # Your STL Courts Response Web Application
```

Comment:
Can you update all of these to use npm scripts only? That way there is no global dependency on a particular version of angular-cli. i.e. `npm start`, `npm run build`, `npm run ng help`, `npm test`, etc.